src/Raven.Server/Dashboard/MachineResourcesNotificationSender.cs

```diff
@@ -83,7 +84,7 @@ public static MachineResources GetMachineResources(SmapsReader smapsReader)
                 var memInfo = MemoryInformation.GetMemoryInfo(smapsReader);
                 var isLowMemory = LowMemoryNotification.Instance.IsLowMemory(memInfo);
                 var workingSet = PlatformDetails.RunningOnLinux
-                    ? MemoryInformation.GetRssMemoryUsage(currentProcess.Id) - memInfo.SharedCleanMemory.GetValue(SizeUnit.Bytes)
+                    ? MemoryInformation.GetRssMemoryUsage(currentProcess.Id) - memInfo.AvailableWithoutTotalCleanMemory.GetValue(SizeUnit.Bytes)
```

Comment:
@aviviadi I want to double check this change. Can you confirm?

Reply:
Agree, this is too complex for an informative only value, let's drop it.